src/commands/lint/__test__/lint.test.ts

```diff
@@ -0,0 +1,45 @@
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+
+import { lintImageryPath, lintPath } from '../lint.s3.paths.js';
+
+describe('lintImageryPaths', () => {
+  it('Should Fail - Incorrect Bucket Name', () => {
+    assert.throws(() => {
+      lintPath('s3://n-imagery/auckland/auckland_2012_0.075m/rgb/2193/');
```

Comment:
Does AWS have a set of test buckets which will always be empty? It's not a stretch that someone will create a bucket called "n-imagery", which would mean this test would fail for the wrong reason.

Reply:
This is specific to these buckets, we are trying to ensure that the imagery folder paths are consistent. while allowing for expansion later into per bucket folder paths.

Reply:
Yep, just realised that was the case after looking at the error checking.